i18n/chs/out/commands.i18n.json

```diff
@@ -0,0 +1,4 @@
+{
+  "autolispext.help.commands.openWebHelp": "",
```

Comment:
it looks like this is empty for `chs`

Reply:
Well, now maybe you can see why I am generally very confused about what I am supposed to do beyond ENU.

The **package.nls.th-tw.json** should have an entry for the context menu itself from a prior PR. I am guessing the localization your looking at now relates to the command palette and was apparently not generated in that prior PR.  Note, this utility made no attempt at managing the package.nls files since those did not match the structure and were not tied to *.ts files.

With all that said, this question only validates my uncertainty. Do you actually want me to go populate all of the missing language values for _autolispext.help.commands.openWebHelp_?

Reply:
`Do you actually want me to go populate all of the missing language values for autolispext.help.commands.openWebHelp?`
I think it is related to our objections. If we want to support the localized string when developing, it needs to populate for all languages. Instead if we don't want to support the localized version, and before the official release there is professional localization team to translate all the string in the `ENU `folder and test them.

I suggest we only add the strings in the `ENU `folder and `package.nls.json`, then we have more time to do other enhancement.

Reply:
Then I would say the current state of this probably represents a "normal" developer state because it does have all the ENU strings populated. The "other" languages do have their keys loaded into the JSON files, but the values are just empty strings. Which should make it really easy for the localization team to identify what they need to do work on.

Reply:
Thank you! I got it.